.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@ repos:
   - repo: https://github.com/norwoodj/helm-docs
     rev: v1.8.1
     hooks:
-      - id: helm-docs
+      - id: helm-docs-built
```

Comment:
requires a `brew install go` on the host instead of helm-docs itself but at least the helm-docs version is the one specified here